packages/video_player/video_player/lib/video_player.dart

```diff
@@ -348,9 +348,11 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
         break;
     }
 
-    if (videoPlayerOptions?.mixWithOthers != null) {
-      await _videoPlayerPlatform
-          .setMixWithOthers(videoPlayerOptions!.mixWithOthers);
+    if (videoPlayerOptions == null) {
```

Comment:
In order to initialise AVAudioSession on iOS by default, on Android it has no effect. 

This was previously handled by **initialize** method of **FLTVideoPlayerPlugin** and it causes
problem for simultaneous playback of audio sources. The same session will be correctly initiated
but with more flexibility.